### PR TITLE
Selenium tests: Fix discrepencies

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
I didn't expect the PR to be that big, sorry the for the long review. The commit list would be a good summary of the changes. But to sum up: we have a common style among the Selenium tests. We also shouldn't have duplication anymore.

r? @npark-mozilla

File: test/selenium/app_test.js [hunk 1]
```diff
@@ -7,15 +7,15 @@ var suiteBuilder = new SuiteBuilder('Github.io webapp', HOST_URL);
 
 suiteBuilder.build((app) => {
 
-  describe('open the web app', () => {
+  describe('open the web app', function() {
```
Comment (nojunpark) [inline test/selenium/app_test.js:11]: ah, so you're removing the arrow notation from the mocha scaffolding?

Comment (JohanLorenzo) [inline test/selenium/app_test.js:11]: Yup, you convinced me :smiley:

File: test/selenium/lib/view/set_up/view.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const View = require('../view');
+const PASSWORDS = require('../../passwords.json');
+
+
+function SetUpView() {
+  View.apply(this, arguments);
+}
+
+SetUpView.prototype = Object.assign({
+
```
Comment (nojunpark) [inline test/selenium/lib/view/set_up/view.js:12]: oh, i didn't know we can define member functions like this as well.  should remember that

File: test/selenium/lib/view/sign_in/view.js [hunk 3]
```diff
@@ -1,40 +1,37 @@
...
+      .then(() => this.alertMessage());
   },
 
-  _submitPassword: function(password) {
-    return this.accessors.password.sendKeys(password)
-    .then(() => this.accessors.submitButton.click());
+  _submitPassword(password) {
+    return this.accessor.passwordField.sendKeys(password)
+      .then(() => this.accessor.submitButton.click());
   },
```
Comment (nojunpark) [inline test/selenium/lib/view/sign_in/view.js:26]: just curious, do we have a waitfor method that can be used after this to check for the UI change?

File: test/selenium/lib/view/signed_in/accessor.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+const Accessor = require('../accessor');
+
+
+function SignedInPageAccessor() {
+  Accessor.apply(this, arguments);
+}
+
+SignedInPageAccessor.prototype = Object.assign({
+  get signOutButton() {
```
Comment (nojunpark) [inline test/selenium/lib/view/signed_in/accessor.js:11]: oh, is this the one that can be used after submitting the password?

File: test/selenium/lib/view/sign_in/view.js (hunk 3 above)
Comment (JohanLorenzo) [inline test/selenium/lib/view/sign_in/view.js:26]: We have them in `successLogin()` and `failureLogin()`. Each one makes sure we're on the next page or we display the alert

File: test/selenium/lib/view/signed_in/accessor.js (hunk 4 above)
Comment (JohanLorenzo) [inline test/selenium/lib/view/signed_in/accessor.js:11]: Yes. This button is on the page showed once you're signed in

**Comment by nojunpark** [comment]
nice cleanup, looks good to me r+

**Pull request merged**